src/basic/build.h

```diff
@@ -127,10 +127,12 @@
 #define _KMOD_FEATURE_ "-KMOD"
 #endif
 
-#ifdef HAVE_LIBIDN
-#define _IDN_FEATURE_ "+IDN"
+#if defined(HAVE_LIBIDN2)
+#  define _IDN_FEATURE_ "+IDN2"
+#elif defined(HAVE_LIBIDN)
+#  define _IDN_FEATURE_ "+IDN"
 #else
-#define _IDN_FEATURE_ "-IDN"
+#  define _IDN_FEATURE_ "-IDN"
```

Comment:
hmm, so far we genreated + and - tags for all deps. Now we will not create any IDN tag if we generate +IDN2, and we will never generate -IDN2... I don't think we need to be briefer than necessary here: i'd just generate "+IDN -IDN2", "-IDN +IDN2" or "-IDN -IDN2" depending on the possible values